pkg/doc/jose/jwe.go

```diff
@@ -64,28 +64,16 @@ type rawJSONWebEncryption struct {
 type marshalFunc func(interface{}) ([]byte, error)
 
 // Serialize serializes the given JWE into JSON as defined in https://tools.ietf.org/html/rfc7516#section-7.2.
+// The full serialization syntax is used. If there is only one recipient, then the flattened syntax is used.
```

Comment:
why flattened instead of compact?

Reply:
compact is not a JSON format, while full and flattened are in JSON

Reply:
Two reasons...
1. Using compact instead of flattened isn't a lossless operation - it would discard the unprotected header and AAD values
2. To match what go-jose does. Go-jose's full serialization method automatically uses flattened syntax when there's only one recipient.

Reply:
we should have a followup to also support compact.